docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -10,6 +10,9 @@ The following figure illustrates flex container terminology and alignment axes,
 
 ![Flex container terminology](media/flex-container.jpg)
 
+> [!NOTE]
+> FlexContainer is deprecated in API8 and will be removed in API10. Please use FlexLayout instead!
```

Comment:
@pkalota Just I have a doubt.
Here FlexContainer refers to?

a class ?? or 

As per definition: A flex container is a NUI container control that contains items. 


can we say: 

> [!NOTE]
> FlexContainer is deprecated in API level 8 and will be removed in API level 10. Please use FlexLayout instead!